lib/class-wp-duotone-gutenberg.php

```diff
@@ -712,6 +712,15 @@ private static function enqueue_custom_filter( $filter_id, $duotone_selector, $f
 	 * @param string $filter_value     The filter CSS value. e.g. 'url(#wp-duotone-blue-orange)' or 'unset'.
 	 */
 	private static function enqueue_global_styles_preset( $filter_id, $duotone_selector, $filter_value ) {
+		if ( ! array_key_exists( $filter_id, self::$global_styles_presets ) ) {
+			$error_message = sprintf(
+				/* translators: %s: duotone filter ID */
+				__( 'The duotone id "%s" is not registered in theme.json settings', 'gutenberg' ),
+				$filter_id
+			);
+			trigger_error( $error_message );
```

Comment:
Trigger error? Let's use `_doing_it_wrong` instead.

Reply:
I was under the impression that `_doing_it_wrong` was used for developers calling functions incorrectly. The function is being called correctly, but the theme.json it's reading from has bad data. Is it still the right choice in this case?